freeipa.spec.in

```diff
@@ -1467,7 +1467,7 @@ fi
 %dir %{_localstatedir}/lib/ipa
 %attr(700,root,root) %dir %{_localstatedir}/lib/ipa/backup
 %attr(700,root,root) %dir %{_localstatedir}/lib/ipa/gssproxy
-%attr(700,root,root) %dir %{_localstatedir}/lib/ipa/sysrestore
+%attr(711,root,root) %dir %{_localstatedir}/lib/ipa/sysrestore
```

Comment:
`711` is not sufficient, because IPA needs to read the contents of sysrestore files in the `/var/lib/ipa/sysrestore` directory, not just list their filenames.

Reply:
This grants permission to enter the directory. The sysrestore.* files are already mode 755 so they are readable by anyone.

You can confirm with: 
    $ cat /var/lib/ipa/sysrestore/sysrestore.index /var/lib/ipa/sysrestore/sysrestore.state

Reply:
Ah, my bad, the `x` bit is sufficient.